### PR TITLE
Feature/sassdoc

### PR DESCRIPTION
**Pull request by elessar-ch**
Add sassdoc capabilities to the build

### CODE REVIEW

**Comment by elessar-ch** [comment]
image
Merge? @davidknezic @FKSI

File: docs/hbs-partials/sassdoc-template.hbs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{{#> page}}
+  {{#*inline "content"}}
+    <h1>{{comment.context.name}}</h1>
+    <p>{{comment.description}}</p>
+    <p>Type: {{comment.context.type}}</p>
+    {{#if comment.context.code}}
+    <div>
```
Comment (FKSI) [inline docs/hbs-partials/sassdoc-template.hbs:7]: Can we display source code only for mixins? I mean is showing source code for every component something relevant?

Comment (elessar-ch) [reply docs/hbs-partials/sassdoc-template.hbs:7]: Yeah, we can do that using custom annotations, which are pretty easy to do.
I think we'll figure out how best to present sassdoc over some time.
Let's track that with an issue.

Comment (FKSI) [reply docs/hbs-partials/sassdoc-template.hbs:7]: Track with #6

**Review by FKSI — APPROVED** [review]

**Pull request merged**